metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppetlabs-provision",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
The version bump should be done in a separate PR which includes a changelog once this is merged.